src/Keboola/DbExtractor/Extractor/Extractor.php

```diff
@@ -60,11 +62,11 @@ public function __construct(array $parameters, array $state = [], ?Logger $logge
         $this->dbParameters = $parameters['db'];
 
         $proxy = new RetryProxy(
-            $this->logger,
-            RetryProxy::DEFAULT_MAX_TRIES,
-            RetryProxy::DEFAULT_BACKOFF_INTERVAL,
-            [PDOException::class]
+            new SimpleRetryPolicy(5),
```

Comment:
Tohle není to samé. Předtím to retryovalo jen PDOException a teď afaik všechno!

Reply:
mělo by to už být stejný, potřeboval jsem ještě předtím fixnout:
https://github.com/keboola/retry/pull/12